internal/compiler/widgets/fluent-base/styling.slint

```diff
@@ -14,12 +14,12 @@ export global Typography {
     out property <int> semibold-font-weight: 600;
 
     out property <TextStyle> body: {
-        font-size: 14px,
+        font-size: 14 * 0.0769rem,
```

Comment:
Looks like the reference here is 13px. Didn't you  want 14 instead? (0.0714) I see 14px in the Typography section in the Fluent 2 Figma designs.

Reply:
Yes 14, but with 1/14 the font size was to small but 1/13 works.

Reply:
Ok :)